### PR TITLE
AC: support ppocr_recognition model

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/adapters/README.md [hunk 1]
```diff
@@ -79,6 +79,9 @@ AccuracyChecker supports following set of adapters:
   * `labels` - optional, list of supported tokens for decoding raw labels (Optional, default configuration is ascii charmap, this parameter ignored if you have decoding part in the model).
   * `eos_index` - index of end of string token in labels. (Optional, default 2, ignored if you have decoding part in the model).
   * `to_lower_case` - allow converting decoded characters to lower case (Optional, default is `True`).
+* `ppocr` - converting PaddlePaddle CRNN-like model output to `CharacterRecognitionPrediction`.
+  * `vocabulary_file` - file with recogniton symbols for decoding.
+  `remove_duplicates` - allow removement of duplicated symbols (Optional, default value - `True`).
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/adapters/README.md:84]:
```suggestion
  *  `remove_duplicates` - allow removement of duplicated symbols (Optional, default value - `True`).
```

File: tools/accuracy_checker/accuracy_checker/adapters/README.md [hunk 2]
```diff
@@ -79,6 +79,9 @@ AccuracyChecker supports following set of adapters:
   * `labels` - optional, list of supported tokens for decoding raw labels (Optional, default configuration is ascii charmap, this parameter ignored if you have decoding part in the model).
   * `eos_index` - index of end of string token in labels. (Optional, default 2, ignored if you have decoding part in the model).
   * `to_lower_case` - allow converting decoded characters to lower case (Optional, default is `True`).
+* `ppocr` - converting PaddlePaddle CRNN-like model output to `CharacterRecognitionPrediction`.
+  * `vocabulary_file` - file with recogniton symbols for decoding.
+  *  `remove_duplicates` - allow removement of duplicated symbols (Optional, default value - `True`).
```
Comment (Anna-Mironova) [inline tools/accuracy_checker/accuracy_checker/adapters/README.md:84]:
```suggestion
  * `vocabulary_file` - file with recogniton symbols for decoding.
  * `remove_duplicates` - allow removement of duplicated symbols (Optional, default value - `True`).
```

File: tools/accuracy_checker/accuracy_checker/adapters/text_recognition.py [hunk 3]
```diff
@@ -335,3 +335,58 @@ def process(self, raw, identifiers, frame_meta):
...
+        params.update({
+            'vocabulary_file': PathField(description='file with decoding labels'),
+            'remove_duplicates': BoolField(
+                optional=True, default=True, description='remove duplications from result string'
+            )
+        })
+        return params
+
+    def configure(self):
+        self.labels_file = self.get_value_from_config('labels_file')
```
Comment (Anna-Mironova) [inline tools/accuracy_checker/accuracy_checker/adapters/text_recognition.py:355]: Maybe the dictionary file is meant here
```suggestion
        self.labels_file = self.get_value_from_config('vocabulary_file')
```

File: tools/accuracy_checker/accuracy_checker/metrics/README.md [hunk 4]
```diff
@@ -23,6 +23,7 @@ Supported representations: `ClassificationAnnotation`, `TextClassificationAnnota
   * `top_k` - the number of classes with the highest probability, which will be used to decide if prediction is correct.
   * `label_map` - the field in annotation metadata, which contains dataset label map (Optional, should be provided if different from default).
 * `character_recognition_accuracy` - accuracy metric for character recognition task. Metric is calculated as a percentage. Direction of metric's growth is higher-better. Supported representations: `CharacterRecognitionAnnotation`, `CharacterRecognitionPrediction`.
+    * `remove_spaces` - allow removement spaces from reference and predicted strings (Optional, default - `False`).
```
Comment (Anna-Mironova) [inline tools/accuracy_checker/accuracy_checker/metrics/README.md:26]:
```suggestion
  * `label_map` - the field in annotation metadata, which contains dataset label map (Optional, should be provided if different from default).
* `character_recognition_accuracy` - accuracy metric for character recognition task. Metric is calculated as a percentage. Direction of metric's growth is higher-better. Supported representations: `CharacterRecognitionAnnotation`, `CharacterRecognitionPrediction`.
  * `remove_spaces` - allow removement spaces from reference and predicted strings (Optional, default - `False`).
```

**Review by Anna-Mironova — APPROVED** [review]

**Pull request merged**